sdk/spring/spring-cloud-azure-supported-spring.json

```diff
@@ -28,14 +28,30 @@
     "releaseStatus" : "SNAPSHOT",
     "snapshot" : true,
     "supportStatus" : "TODO",
-    "spring-boot-version" : "2.7.3-SNAPSHOT",
+    "spring-boot-version" : "2.7.4-SNAPSHOT",
     "spring-cloud-version" : "2021.0.3"
   },
   {
     "current" : true,
     "releaseStatus" : "GENERAL_AVAILABILITY",
     "snapshot" : false,
     "supportStatus" : "SUPPORTED",
+    "spring-boot-version" : "2.7.3",
+    "spring-cloud-version" : "2021.0.3"
+  },
+  {
+    "current" : false,
+    "releaseStatus" : "SNAPSHOT",
+    "snapshot" : true,
+    "supportStatus" : "END_OF_LIFE",
+    "spring-boot-version" : "2.7.3-SNAPSHOT",
+    "spring-cloud-version" : "2021.0.3"
+  },
+  {
+    "current" : false,
+    "releaseStatus" : "GENERAL_AVAILABILITY",
+    "snapshot" : false,
+    "supportStatus" : "END_OF_LIFE",
```

Comment:
Why `2.7.2` is `END_OF_LIFE`? I think we should support `2.7.2`.

Does `END_OF_LIFE` mean we do not run IT for this version?
If `END_OF_LIFE` means we do not run IT for this version, I suggest using another name instead of `END_OF_LIFE`

Reply:
Is there any documentation that explains this json file?

Reply:
There is no documentation to explain for now. 
As for changing the name of `supportStatus`, it's a good suggestion, we could talk about it after @saragluna back from vacation.